lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -66,7 +66,7 @@
 }
 
 .forum-nav-search .icon {
-  @include font-size(12);
+  font-size: 12px;
```

Comment:
We should use one of our defined size variables, or use rem/em. Also 12px is tiny. 16px should be our minimum. If we absolutely need smaller, 14px.